crypto/ec/ecp_sm2p256.c

```diff
@@ -0,0 +1,791 @@
+/*
+ * Copyright 2023 The OpenSSL Project Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License 2.0 (the "License").  You may not use
+ * this file except in compliance with the License.  You can obtain a copy
+ * in the file LICENSE in the source distribution or at
+ * https://www.openssl.org/source/license.html
+ *
+ */
+
+/*
+ * SM2 low level APIs are deprecated for public use, but still ok for
+ * internal use.
+ */
+#include "internal/deprecated.h"
+
+#include <string.h>
+#include <openssl/err.h>
+#include "crypto/bn.h"
+#include "ec_local.h"
+#include "internal/constant_time.h"
+
+#if defined(__GNUC__)
+#define ALIGN32 __attribute((aligned(32)))
+#define ALIGN64 __attribute((aligned(64)))
```

Comment:
Add a space between the `#` and the `define` here and below.
Our coding style indents nested preprocessor directives.